lib/raygun/configuration.rb

```diff
@@ -143,7 +146,8 @@ def initialize
         breadcrumb_level:              :info,
         record_raw_data:               false,
         send_in_background:            false,
-        error_report_send_timeout:     10
+        error_report_send_timeout:     10,
+        track_retried_sidekiq_jobs:    true
```

Comment:
Is `true` a good default value for this? I am not sure how annoying it is to have retries reported, but it is good that it can be disabled if required.

Reply:
I think so - rationale being it's better to log more errors initially and then turn off any that are noisy rather than silently swallow them